### PR TITLE
Support activerecord-import 1.x.x [transport_all]

### LINKED ISSUE #12: transport_all rename-strategy is not working with activerecord-import 1.x.x
ErrorTrace
```
Traceback (most recent call last):
	44: from bin/rails:4:in `<main>'
	43: from xxx/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `require'
	42: from xxx/activesupport-5.2.2/lib/active_support/dependencies.rb:257:in `load_dependency'
	41: from xxx/activesupport-5.2.2/lib/active_support/dependencies.rb:291:in `block in require'
	40: from xxx/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
	39: from xxx/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
	38: from xxx/bootsnap-1.4.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:83:in `register'
	37: from xxx/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
	36: from xxx/bootsnap-1.4.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
	35: from xxx/railties-5.2.2/lib/rails/commands.rb:18:in `<main>'
	34: from xxx/railties-5.2.2/lib/rails/command.rb:46:in `invoke'
	33: from xxx/railties-5.2.2/lib/rails/command/base.rb:65:in `perform'
	32: from xxx/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	31: from xxx/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	30: from xxx/thor-0.20.3/lib/thor/command.rb:27:in `run'
	29: from xxx/railties-5.2.2/lib/rails/commands/runner/runner_command.rb:41:in `perform'
	28: from xxx/railties-5.2.2/lib/rails/commands/runner/runner_command.rb:41:in `eval'
	27: from xxx/railties-5.2.2/lib/rails/commands/runner/runner_command.rb:41:in `<main>'
	26: from xxx/xxx/dwh/batch_job.rb:28:in `run'
	25: from xxx/redshift-connector-7.2.1/lib/redshift_connector/connector.rb:170:in `execute'
	24: from xxx/redshift-connector-7.2.1/lib/redshift_connector/connector.rb:180:in `import'
	23: from xxx/redshift-connector-7.2.1/lib/redshift_connector/importer/rebuild_rename.rb:22:in `execute'
	22: from xxx/redshift-connector-7.2.1/lib/redshift_connector/importer/rebuild_rename.rb:35:in `import'
	21: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:38:in `each_batch'
	20: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:52:in `do_each_batch'
	19: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:13:in `each_row'
	18: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:21:in `each_object'
	17: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:21:in `each'
	16: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:23:in `block in each_object'
	15: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:14:in `block in each_row'
	14: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file.rb:15:in `each_row'
	13: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/reader/redshift_csv.rb:24:in `each_row'
	12: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:55:in `block in do_each_batch'
	11: from xxx/redshift-connector-data_file-7.3.1/lib/redshift_connector/abstract_data_file_bundle.rb:39:in `block in each_batch'
	10: from xxx/redshift-connector-7.2.1/lib/redshift_connector/importer/rebuild_rename.rb:36:in `block in import'
	 9: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:528:in `bulk_import'
	 8: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:692:in `import_helper'
	 7: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:750:in `import_with_validations'
	 6: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:802:in `import_without_validations_or_callbacks'
	 5: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:802:in `each_slice'
	 4: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:802:in `each'
	 3: from xxx/activerecord-import-1.0.1/lib/activerecord-import/import.rb:807:in `block in import_without_validations_or_callbacks'
	 2: from xxx/activemodel-5.2.2/lib/active_model/naming.rb:243:in `model_name'
	 1: from xxx/activemodel-5.2.2/lib/active_model/naming.rb:243:in `new'
xxx/activemodel-5.2.2/lib/active_model/naming.rb:152:in `initialize': Class name cannot be blank. You need to supply a name argument when anonymous class given (ArgumentError)
```

### PR DESCRIPTION
**Pull request by k4zy**
resolved: https://github.com/bricolages/redshift-connector/issues/12

### CODE REVIEW

**Comment by k4zy** [comment]
@aamine @KOBA789 What do you think about this solution?

File: lib/redshift_connector/importer/rebuild_rename.rb [hunk 1]
```diff
@@ -14,12 +14,11 @@ def execute(bundle)
       tmp_table = "#{dest_table}_new"
       old_table = "#{dest_table}_old"
 
-      tmp_dao = @dao.dup
-      tmp_dao.table_name = tmp_table
+      @dao.table_name = tmp_table
```
Comment (aamine) [inline lib/redshift_connector/importer/rebuild_rename.rb]: This is destructive operation, naming tmp_dao such like the following code is better.
```
tmp_dao = @dao.dup
self.class.const_set(:TmpDAO_00001, tmp_dao)   # give the tmp_dao the const path
tmp_dao.name   # force tmp_dao to have the class name
```

Comment (k4zy) [reply lib/redshift_connector/importer/rebuild_rename.rb]: 👍  fixed https://github.com/bricolages/redshift-connector/commit/5a945c7c691867b772277e40f663a7454e02364c

**Review by aamine — APPROVED** [review]
Thank you very much, I release this fix now

**Pull request merged**